### DISCUSSION TITLE
Is there a desktop app that supports GPU?

### DISCUSSION THREAD

**Discussion by Vuizur** [openai/whisper#721] [Q&A]
Hello everyone,

I have searched for it, but couldn't seem to find anything. Is there an easily installable Whisper-based desktop app that has GPU support? Thanks!

**Comment by Sharrnah** [comment]
I normally don't want to advertise myself. But you can have a look at my project here (see the Quickstart section):
https://github.com/Sharrnah/whispering#readme

I just finished the first pre-releases of a native UI (windows only for the moment).
It downloads and runs the python project in the background.

<img src=https://github.com/Sharrnah/whispering-ui/raw/main/doc/images/speech2text.png>

**Reply by Vuizur** [reply]
I will try it, thank you very much for the answer and development!

**Comment by mayeaux** [comment]
Can also use my app if you want to run it via a webapp from your browser. Could probably build it for desktop via Electron but not a big priority atm: https://github.com/mayeaux/generate-subtitles

**Reply by Vuizur** [reply]
Thank you very much, I will look at it.

**Comment by fleek** [comment]
if you installed the GPU versions of tensorflow and pytorch, you will get GPU support automatically.

https://pytorch.org/get-started/previous-versions/

but you also have to specify to use Cuda in whisper
 wmodel = whisper.load_model(ms, download_root="models", device=dv) where dv = 'cpu' or 'cuda' only working for nvidia gpus, I have not tried RocM or directml

**Reply by Vuizur** [reply]
Thanks!
